implementations/scripts/Metric.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
+
+
+class Metric():
+    """
+    Create a dataframe from data-list, which is the data
+    fetched by Perceval
+
+    :param data_list: A list of dictionaries.
+        Each element is a Perceval dictionary, obtained from a JSON
+        file or from Perceval directly.
+    """
+
+    def __init__(self, data_list):
+        data_list_flattened = Metric._flatten_data(data_list)
```

Comment:
This should be extensible by child classes, and therefore, `self._flatten_data()` (or maybe `class._flatten_data()`)